src/newViews/PillarAnalysis/AnalyticalStatementInput/index.tsx

```diff
@@ -168,6 +168,10 @@ function AnalyticalStatementInput(props: AnalyticalStatementInputProps) {
         [handleAnalyticalEntryDrop],
     );
 
+    const handleIncludeInReportChange = useCallback(() => {
+        onFieldChange((oldVal?: boolean) => !oldVal, 'includeInReport' as const);
+    }, [onFieldChange]);
+
```

Comment:
Why do we need this? The input should have done this by itself

Reply:
Its a button. Should I change the name?

Reply:
Should have used a checkbox component instead.